### ISSUE TITLE
Error occurs when opening a lot of tabs (in a fast manner)

### ISSUE THREAD

**Issue by bsclifton** [brave/browser-laptop#9721]
- Did you search for similar issues before submitting this one?
Yes

- Describe the issue you encountered:
With a fresh session, I middle clicked the default tiles that show up on new page, hoping to lots of tabs to my session.  Unfortunately, this caused a crash:
```
An uncaught exception occurred in the main process Uncaught Exception:
TypeError: parsedUrl.query.split is not a function
    at SearchIndex.tokenizeInput (/Users/clifton/Documents/browser-laptop/app/common/lib/siteSuggestions.js:95:44)
    at /Users/clifton/Documents/browser-laptop/node_modules/bloodhound-js/lib/search_index.js:36:37
    at Array.forEach (<anonymous>)
    at Object.each (/Users/clifton/Documents/browser-laptop/node_modules/bloodhound-js/lib/utils.js:52:16)
    at SearchIndex.add (/Users/clifton/Documents/browser-laptop/node_modules/bloodhound-js/lib/search_index.js:32:7)
    at Bloodhound.add (/Users/clifton/Documents/browser-laptop/node_modules/bloodhound-js/lib/bloodhound.js:110:16)
    at add (/Users/clifton/Documents/browser-laptop/app/common/lib/siteSuggestions.js:113:12)
    at urlBarSuggestionsReducer (/Users/clifton/Documents/browser-laptop/app/browser/reducers/urlBarSuggestionsReducer.js:22:9)
    at reducers.reduce (/Users/clifton/Documents/browser-laptop/js/stores/appStore.js:380:24)
    at Array.reduce (<anonymous>)
```

- Platform (Win7, 8, 10? macOS? Linux distro?):
macOS

- Brave Version (revision SHA):
0.17.9

- Steps to reproduce:
    1. With a new session, open Brave and ensure you're on the new tab page
    2. Middle click or cmd/ctrl + click the new tab icons for brave.com and twitter.com/brave as fast as you can
    3. Keep going until you have several tab pages worth of tabs
    
- Actual result:
Sometimes, a crash happens (text shown above)

- Expected result:
No crash- all tabs should load

- Will the steps above reproduce in a fresh profile? If not what other info can be added?
yes

- Is this an issue in the currently released version?
no

- Can this issue be consistently reproduced?
yes

- Extra QA steps:
    1.
    2.
    3.

- Screenshot if needed:

- Any related issues:

**Comment by diracdeltas** [comment]
should be fixed in 0.17.x by https://github.com/brave/browser-laptop/commit/31b5af3b73f20ded05f030bbbb1704a3d9b059ff

**Comment by luixxiul** [comment]
the browser does not crash but loading a hundred takes minutes and not only the browser but also the OS gets stucked until they are loaded.
